Packs/TheHiveProject/Integrations/TheHiveProject/TheHiveProject.py

```diff
@@ -0,0 +1,730 @@
+            'severity': case['severity'],
+            'rawJSON': json.dumps(case)
+        }
+        incidents.append(incident)
+        last_timestamp = case['createdAt'] if case['createdAt'] > last_timestamp else last_timestamp
+    demisto.setLastRun({"timestamp": str(last_timestamp)})
+    return incidents
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    params = demisto.params()
+    args = demisto.args()
+    api_key = params['apiKey']
+    url = params.get('url')
+    base_url = f'{url}/api'
+    verify_certificate = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    mirroring = params.get('mirroring', None).title() if params.get('mirroring', None) else None
+    mirroring = None if mirroring == 'Disabled' else mirroring
+
+    command = demisto.command()
+
+    command_map = {
+        'thehive-list-cases': list_cases_command,
+        'thehive-get-case': get_case_command,
+        'thehive-search-cases': search_cases_command,
+        'thehive-update-case': update_case_command,
+        'thehive-create-case': create_case_command,
+        'thehive-remove-case': remove_case_command,
+        'thehive-get-linked-cases': get_linked_cases_command,
+        'thehive-merge-cases': merge_cases_command,
+        'thehive-get-case-tasks': get_case_tasks_command,
+        'thehive-get-task': get_task_command,
+        'thehive-get-attachment': get_attachment_command,
+        'thehive-update-task': update_task_command,
+        'thehive-list-users': search_users_command,
+        'thehive-get-user': get_user_command,
+        'thehive-create-local-user': create_local_user_command,
+        'thehive-block-user': block_user_command,
+        'thehive-list-observables': list_observables_command,
+        'thehive-create-observable': create_observable_command,
+        'thehive-update-observable': update_observable_command
+    }
+    demisto.debug(f'Command being called is {command}')
+    try:
+        headers = {
+            'Authorization': f'Bearer {api_key}'
+        }
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            headers=headers,
+            proxy=proxy,
+            mirroring=mirroring
+        )
+
+        if command == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            return_results(result)
+
+        elif command == 'fetch-incidents':
+            # Set and define the fetch incidents command to run after activated via integration settings.
+            incidents = fetch_incidents(client)
+            demisto.incidents(incidents)
+
+        elif command == 'get-remote-data':
+            demisto.results(get_remote_data_command(client, args, params))
+
+        elif command == 'update-remote-system':
+            demisto.results(update_remote_system_command(client, args, params))
```

Comment:
```suggestion
            update_results(update_remote_system_command(client, args, params))
```

Reply:
Did you mean `return_results`?
I do not see `update_results` throughout the repo.

Reply:
Yes, you're right. My bad